docs/acp/user/app-manifest.md

```diff
@@ -69,33 +69,43 @@ For the manifest, make sure to follow these conditions:
+| labels           | An array of strings. Displays label icons on the application detail page according to the label. Possible values:<ul><li>Silver Partner</li><li>Gold Partner</li><li>New</li><li>Popular</li><li>Free Trial</li></ul>                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |See *labels example* under this table.   |
+| resources        | An array of objects represented as application resources (links, documents). The resource object has the following keys:<ul><li>title—name of the resource</li><li>url—full URL to the resource</li>type—type of the resource. Optional. The type affects a displayed icon. Possible values:<ul><li>internal-documentation</li><li>homepage</li><li>user-documentation</li><li>developer-documentation</li><li>release-notes</li></ul><li>fileType—type of the resource file (in case if the url leads to a file and if `type=internal-documentation`). Optional. The files are displayed directly in the AppCatalog, without redirecting to another page. Possible values:<ul><li>pdf</li><li>markdown</li></ul></ul> |See *resources example* under this table.   |
+| messages         | Optional. An array of objects represented as application messages. The message object has following keys: <ul><li>`type` - Mandatory. Message type (available types: `text`).</li><li>`data` - Mandatory. Message data.</li></ul>                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      | See *message example* under this table.
```

Comment:
What is message data? is it text or JSON? if it's an object what properties are needed? Would be nice to have an example in the example section.

Reply:
@mharacewiat @Serhii-SPRYKER there is also the picture at the end of the page which shows where the parts of manifests are used. So please add a picture with the example where the new feature is used (config page -> dialogs)